x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -134,6 +260,23 @@ export default function ({ getService }: FtrProviderContext) {
           },
           getBody: () => undefined,
         },
+        {
+          method: 'post',
+          info: 'create item',
+          path: EXCEPTION_LIST_ITEM_URL,
+          getBody: () => exceptionsGenerator.generateHostIsolationExceptionForCreate(),
+        },
+        {
+          method: 'put',
+          info: 'update item',
+          path: EXCEPTION_LIST_ITEM_URL,
+          getBody: () =>
+            exceptionsGenerator.generateHostIsolationExceptionForUpdate({
+              id: existingExceptionData.artifact.id,
+              item_id: existingExceptionData.artifact.item_id,
+              _version: existingExceptionData.artifact._version,
+            }),
+        },
```

Comment:
I don't think you need these - they are already included in line 222 from the prior array of api calls defined on line 64

Reply:
ah, missed the spread at the top. will remove in follow up PR.